client/blocks/reader-subscription-list-item/connected.jsx

```diff
@@ -38,7 +38,6 @@ class ConnectedSubscriptionListItem extends React.Component {
 	};
 
 	componentDidMount() {
-		this.props.onLoad();
```

Comment:
no point of measuring right when it mounts -- that happens anyway. What matters is to measure each time the props change